lib/puppet/vendor/pathspec/lib/pathspec/gitignorespec.rb

```diff
@@ -145,15 +145,14 @@ def match(path)
   end
 
   def translate_segment_glob(pattern)
-    """
-    Translates the glob pattern to a regular expression. This is used in
-    the constructor to translate a path segment glob pattern to its
-    corresponding regular expression.
-
-    *pattern* (``str``) is the glob pattern.
-
-    Returns the regular expression (``str``).
-    """
+    # Translates the glob pattern to a regular expression. This is used in
+    # the constructor to translate a path segment glob pattern to its
+    # corresponding regular expression.
+    #
+    # *pattern* (``str``) is the glob pattern.
+    #
+    # Returns the regular expression (``str``).
+    #
```

Comment:
Do we care about the fact this is from a vendored project?

Reply:
I completely missed that when I was updating things. Became snowblind to the paths involved. I'll remove that change. We shouldn't be manually updating files in there individually.

Reply:
I thought as much, but I wanted to check :-D